extensions/wikia/DesignSystem/DesignSystemHooks.class.php

```diff
@@ -0,0 +1,59 @@
+<?php
+class DesignSystemHooks {
+
+	const DESIGN_SYSTEM_DIR = __DIR__ . '/bower_components/design-system/dist';
+
+	/**
+	 * @desc Adds Design System styles to all Oasis pages
+	 *
+	 * @param OutputPage $out
+	 *
+	 * @return bool true
+	 */
+	public static function onBeforePageDisplay( $out, &$skin ) {
```

Comment:
No need for the `&` on `$skin`. Objects in PHP are (effectively) passed by reference anyway.